xapian-core/cluster/cluster.cc

```diff
@@ -110,18 +121,16 @@ TermListGroup::add_document(const Document &document) {
 }
 
 void
-TermListGroup::add_documents(DocumentSource &docs) {
+TermListGroup::add_documents(MSetDocumentSource docs) {
```

Comment:
Okay great! I'll add that in right away. I guess that removes two classes from the public API, so it does help 👍